Bio/SeqIO/GckIO.py

```diff
@@ -229,7 +229,8 @@ def GckIterator(source):
             raise ValueError("GCK files must be opened in binary mode.") from None
 
     try:
-        return _parse(handle)
+        records = _parse(handle)
+        yield from records
```

Comment:
I would have gone for:

```python
yield from _parse(handle)
```

Reply:
That is fine too. Feel free to change it.